apps/angular/src/app/features/auth/login/login.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, DestroyRef, inject } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { UserService } from '@js-camp/angular/core/services/user.service';
+import { catchFormErrors } from '@js-camp/angular/core/utils/catch-form-error';
+import { ControlsOf } from '@js-camp/angular/core/utils/types/controls-of';
+import { AppValidationError } from '@js-camp/core/models/app-error';
+import { Login } from '@js-camp/core/models/auth/login';
+import { BehaviorSubject, catchError, finalize, first, throwError } from 'rxjs';
+
+import { MIN_PASSWORD_LENGTH } from '../utils/constants';
+
+type LoginForm = ControlsOf<Login>;
+
+/** Login page. */
+@Component({
+	selector: 'camp-login',
+	templateUrl: './login.component.html',
+	styleUrls: ['../auth.css'],
+})
+export class LoginComponent {
+	/** Loading status. */
+	protected readonly isLoading$ = new BehaviorSubject(false);
+
+	/** Login form. */
+	protected readonly loginForm: FormGroup<LoginForm>;
+
+	/** Common global form errors. */
+	protected readonly commonErrors$ = new BehaviorSubject('');
+
+	/** Form builder. */
+	private readonly formBuilder = inject(NonNullableFormBuilder);
+
+	/** User service. */
+	private readonly userService = inject(UserService);
+
+	/** Destroy ref. */
+	private readonly destroyRef = inject(DestroyRef);
+
+	/** Router. */
+	private readonly router = inject(Router);
+
+	public constructor() {
+		this.loginForm = this.initLoginForm();
+	}
+
+	/** Handle 'submit' of the login form. */
+	protected login(): void {
```

Comment:
We can match the name with docs to make it more clear. Also, we usually use `on` prefix for template handlers
```suggestion
	protected onSubmit(): void {
```